### PR TITLE
screenshot implementation

### PR DESCRIPTION
**Pull request by svaleria**
- after scene screenshot for failed tests

### CODE REVIEW

File: 3.Tests/UI/Hooks/Hooks.cs [hunk 1]
```diff
@@ -41,12 +44,6 @@ public static void CleanUp()
         APIScripts.RemoveAllProjects();
     }
 
-    [AfterScenario]
-    public void SessionDisposal()
```
Comment (sebasonti-jalauni) [inline 3.Tests/UI/Hooks/Hooks.cs:45]: There is no need to remove this hook, the CaptureScreenshot hook should only do that, not disposing the driver session as well.

File: 3.Tests/UI/Hooks/Hooks.cs [hunk 2]
```diff
@@ -1,6 +1,9 @@
 ﻿using System;
+using System.Linq;
 using System.Text.Json;
+using OpenQA.Selenium;
 using RestSharp;
+using Sprache;
```
Comment (sebasonti-jalauni) [inline 3.Tests/UI/Hooks/Hooks.cs:6]: What is this package needed for? if it has to be installed, you should also commit the .csproj with the respective package reference

File: 3.Tests/UI/Hooks/Hooks.cs [hunk 3]
```diff
@@ -61,6 +58,23 @@ public void CreateProject()
...
+    {
+        if (_scenarioContext.TestError != null)
+        {
+            Screenshot image = ((ITakesScreenshot)GenericWebDriver.Instance).GetScreenshot();
+            string fileName = $"{_scenarioContext.ScenarioInfo.Title}_{DateTime.Now}";
+            fileName = string.Join(" ", fileName.Split().Select(word => char.ToUpper(word[0]) + word.Substring(1)));
+            fileName = fileName.Replace(" ", "");
+            fileName = fileName.Replace("/", "");
+            fileName = fileName.Replace(":", "");
+            image.SaveAsFile($"../../../Assets/{fileName}.png", ScreenshotImageFormat.Png);
```
Comment (sebasonti-jalauni) [inline 3.Tests/UI/Hooks/Hooks.cs:72]: I think it is better to create a new folder for each test execution, and there you can store the screenshots for the whole test run. You are also missing to attach the screenshot to the LivingDoc report, is it going to be done on another PR?

Comment (svaleria) [reply 3.Tests/UI/Hooks/Hooks.cs:72]: hi Sebastian, yeah in order to touch just the yaml file for the reports it´s going to be in the valeria/reports branch, and for the screenshots just this branch

File: 3.Tests/UI/Hooks/Hooks.cs (hunk 2 above)
Comment (svaleria) [reply 3.Tests/UI/Hooks/Hooks.cs:6]: This library i was using to name the png file, but i ended up using linq. Sprache is used for parsing. As now im not using it, its  deleted

File: 3.Tests/UI/Hooks/Hooks.cs (hunk 1 above)
Comment (svaleria) [reply 3.Tests/UI/Hooks/Hooks.cs:45]: solved

**Review by Schlvf — APPROVED** [review]

**Review by gioacchinogioffre — APPROVED** [review]
